assets/js/base/components/cart-checkout/product-details/style.scss

```diff
@@ -3,6 +3,10 @@
 	list-style: none;
 	margin: 0.5em 0;
 	padding: 0;
+
+	&:last-of-type {
+		margin: 0;
```

Comment:
It was just because the previous `product-details` element has a margin-bottom, and so the margin-top "overlaps" and doesn't actually do anything. I figured we could save a couple of bytes by doing it this way.

Reply:
Oh, but if there is only one product details list, then there is no margin-top, no? I think this is reproducible if you compare a variable subscription (has two details list) with a simple subscription (has one details list). The margin between the product description and the first details list will be different.

Reply:
Ah yea, you're correct. Thanks for pointing this out 🤦🏻‍♂️ fix pushed.